lib/nailed/changes.rb

```diff
@@ -0,0 +1,120 @@
+module Nailed
+
+  #
+  # helpers
+  #
+  def write_changes
+    Config.supported_vcs.each do |vcs|
+      client = const_get(vcs.capitalize).new
+      client.get_change_requests
+    end
+    write_allchangetrends
+  end
+
+  def list_org_repos(client, org)
```

Comment:
What is this function? I know it's used for `nailed --list` but I tried that out and I am thoroughly confused about what it does or what it is supposed to do :confused:

Reply:
It is used to print the repositories of a given github organization. 
So e.g. `nailed --list "kubic-project"` will print you all the repositories of 
kubic-project. 
I guess this is a relic of the past and not used during usual deployment.
You can only access this via the command line.